R/tm_t_crosstable.R

```diff
@@ -352,27 +347,22 @@ srv_t_crosstable <- function(id, datasets, reporter, label, x, y, basic_table_ar
         card <- teal.reporter::TealReportCard$new()
         card$set_name("Cross Table")
         card$append_text("Cross Table", "header2")
-        card$append_text("Filter State", "header3")
```

Comment:
Do we have an issue to add encoding information into tmg modules as I could imagine say waning to know which join we've selected here

Reply:
https://github.com/insightsengineering/teal.modules.general/issues/434